narwhal/storage/src/consensus_store.rs

```diff
@@ -7,33 +7,25 @@ use std::collections::HashMap;
 use store::rocks::{open_cf, DBMap, MetricConf, ReadWriteOptions};
 use store::{reopen, Map, TypedStoreError};
 use tracing::debug;
-use types::{
-    CommittedSubDag, CommittedSubDagShell, ConsensusCommit, ConsensusCommitV2, Round,
-    SequenceNumber,
-};
+use types::{CommittedSubDag, ConsensusCommit, ConsensusCommitV2, Round, SequenceNumber};
 
 /// The persistent storage of the sequencer.
 pub struct ConsensusStore {
     /// The latest committed round of each validator.
     last_committed: DBMap<AuthorityIdentifier, Round>,
-    /// TODO: remove once released to validators
-    /// The global consensus sequence.
-    committed_sub_dags_by_index: DBMap<SequenceNumber, CommittedSubDagShell>,
     /// The global consensus sequence
-    committed_sub_dags_by_index_v2: DBMap<SequenceNumber, ConsensusCommit>,
```

Comment:
Have you tried an upgrade test with this change? I wonder if changing this field name could cause issues with the live store.

Reply:
Yeah, I think we cannot remove tables right now. The canonical way to handle this right now is `allow_unused`. `reopen!` of a rocksdb cannot have missing column family.

Reply:
@arun-koshy I haven't tried any upgrade test - but it was in my plans for sure. In that particular case I don't rename the column family it self but rather the Rust code variable which shouldn't play a role. But I think with the point that also @mwtian makes here I probably need to take care how the reopen should work if I want to do this - otherwise we can protocol config but I would like to leave this as last resort